### PR TITLE
strip operator framework from resource watch and fix bugs

### PR DESCRIPTION
**Pull request by xueqzhan**
[TRT-1054](https://issues.redhat.com/browse/TRT-1054)

### CODE REVIEW

**Comment by xueqzhan** [comment]
/payload-job periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-sdn-upgrade

**Comment by xueqzhan** [comment]
/payload-job periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-sdn-upgrade

**Comment by xueqzhan** [comment]
/hold

Need to change release repo first

**Comment by xueqzhan** [comment]
/payload-job periodic-ci-openshift-release-master-nightly-4.14-e2e-aws-sdn-upgrade

File: pkg/monitor/resourcewatch/storage/git_store.go [hunk 1]
```diff
@@ -31,19 +31,19 @@ type workingSet struct {
 
 func (s *workingSet) isWorkingOn(key string) bool {
 	s.lock.RLock()
-	defer s.lock.RLock()
```
Comment (deads2k) [inline pkg/monitor/resourcewatch/storage/git_store.go:34]: !

**Pull request merged**